public/static/app/perf-bb/PerfBBOutliersCtrl.test.js

```diff
@@ -2,21 +2,55 @@ describe('PerfBBOutliersCtrlTest', () => {
   beforeEach(module('MCI'));
 
   let controller;
+  let $log;
+  let $timeout;
   let scope;
   let format;
 
+  let OutliersDataService;
+
+  let OutlierState;
+  let state;
+
+  let MuteHandler;
+  let mute_handler;
+
+  let Operations;
+
+  let Settings;
+  let Lock;
+  let vm;
+
   const project = 'test-project';
 
   beforeEach(inject(function ($rootScope, $controller, $injector) {
-    // $scope, $window, EvgUiGridUtil, EvgUtil, FORMAT, MDBQueryAdaptor, uiGridConstants,
-    //     STITCH_CONFIG, Stitch,
     scope = $rootScope;
     format = $injector.get('FORMAT');
+    OutliersDataService = $injector.get('OutliersDataService');
+    $log = $injector.get('$log');
+    // $timeout = $injector.get('$timeout');
```

Comment:
[nit] Can this commented out line be removed?

Reply:
:+1: